internal/export/database/export_test.go

```diff
@@ -541,6 +542,109 @@ func TestTravelerKeys(t *testing.T) {
 	}
 }
 
+// TestExcludeRegions ensures excluded regions are excluded.
+func TestExcludeRegions(t *testing.T) {
+	t.Parallel()
+
+	testDB := database.NewTestDatabase(t)
+	ctx := context.Background()
+	now := time.Now()
+
+	// Add a config.
+	inclRegion, exclRegion := "US", "EU"
+	ec := &model.ExportConfig{
+		BucketName:     "bucket-name",
+		FilenameRoot:   "filename-root",
+		Period:         3600 * time.Second,
+		OutputRegion:   inclRegion,
+		From:           now.Add(-24 * time.Hour),
+		ExcludeRegions: []string{exclRegion},
+	}
+	if err := New(testDB).AddExportConfig(ctx, ec); err != nil {
+		t.Fatal(err)
+	}
+
+	// Create a batch for two hours ago to one hour ago.
+	startTimestamp := now.Truncate(time.Hour).Add(-2 * time.Hour)
+	endTimestamp := startTimestamp.Add(time.Hour)
+	eb := &model.ExportBatch{
+		ConfigID:       ec.ConfigID,
+		BucketName:     ec.BucketName,
+		FilenameRoot:   ec.FilenameRoot,
+		StartTimestamp: startTimestamp,
+		EndTimestamp:   endTimestamp,
+		OutputRegion:   ec.OutputRegion,
+		Status:         model.ExportBatchOpen,
+		ExcludeRegions: ec.ExcludeRegions,
+	}
+	if err := New(testDB).AddExportBatches(ctx, []*model.ExportBatch{eb}); err != nil {
+		t.Fatal(err)
+	}
+
+	// Create person in main region.
+	trav := &publishmodel.Exposure{
```

Comment:
this says `trav` but the exposure isn't marked as a traveler

Reply:
my mistake